ex2_hal/athena/equipment_handler/include/rtcmk.h

```diff
@@ -10,335 +10,35 @@
-#define RTCMK_CONTROL_TE_START (_RTCMK_CONTROL_TE_START << 3)
-#define RTCMK_CONTROL_FIE (0x01UL << 4)
-#define _RTCMK_CONTROL_FIE_SHIFT 4
-#define _RTCMK_CONTROL_FIE_MASK 0x10UL
-#define _RTCMK_CONTROL_FIE_DEFAULT 0x00UL
-#define _RTCMK_CONTROL_FIE_DISABLE 0x00UL
-#define _RTCMK_CONTROL_FIE_ENABLE 0x01UL
-#define RTCMK_CONTROL_FIE_DEFAULT (_RTCMK_CONTROL_FIE_DEFAULT << 4)
-#define RTCMK_CONTROL_FIE_DISABLE (_RTCMK_CONTROL_FIE_DISABLE << 4)
-#define RTCMK_CONTROL_FIE_ENABLE (_RTCMK_CONTROL_FIE_ENABLE << 4)
-#define RTCMK_CONTROL_RAM (0x01UL << 5)
-#define _RTCMK_CONTROL_RAM_SHIFT 5
-#define _RTCMK_CONTROL_RAM_MASK 0x20UL
-#define RTCMK_CONTROL_TEST (0x01UL << 6)
-#define _RTCMK_CONTROL_TEST_SHIFT 6
-#define _RTCMK_CONTROL_TEST_MASK 0x40UL
-#define _RTCMK_CONTROL_TEST_DEFAULT 0x00UL
-#define _RTCMK_CONTROL_TEST_DISABLE 0x00UL
-#define _RTCMK_CONTROL_TEST_ENABLE 0x01UL
-#define RTCMK_CONTROL_TEST_DEFAULT (_RTCMK_CONTROL_TEST_DEFAULT << 6)
-#define RTCMK_CONTROL_TEST_DISABLE (_RTCMK_CONTROL_TEST_DISABLE << 6)
-#define RTCMK_CONTROL_TEST_ENABLE (_RTCMK_CONTROL_TEST_ENABLE << 6)
-#define RTCMK_CONTROL_RESET (0x01UL << 7)
-#define _RTCMK_CONTROL_RESET_SHIFT 7
-#define _RTCMK_CONTROL_RESET_MASK 0x80UL
-
-/*******************************************************************************
- ********************************   ENUMS   ************************************
- ******************************************************************************/
-
-typedef enum {
-    RTCMK_RegSec = 0x00,          /**< Seconds register */
-    RTCMK_RegMin = 0x01,          /**< Minutes register */
-    RTCMK_RegHour = 0x02,         /**< Hours register */
-    RTCMK_RegWeek = 0x03,         /**< Day of week register */
-    RTCMK_RegDay = 0x04,          /**< Day of month register */
-    RTCMK_RegMonth = 0x05,        /**< Month register */
-    RTCMK_RegYear = 0x06,         /**< Year register */
-    RTCMK_RegMinAlarm = 0x07,     /**< Alarm minutes register */
-    RTCMK_RegHourAlarm = 0x08,    /**< Alarm hours register */
-    RTCMK_RegWeekDayAlarm = 0x09, /**< Alarm weeks or days register */
-    RTCMK_RegCounter = 0x0A,      /**< Timer counter register  */
-    RTCMK_RegSelect = 0x0B,       /**< Select register */
-    RTCMK_RegFlag = 0x0C,         /**< Flag register */
-    RTCMK_RegControl = 0x0D       /**< Control register */
-} RTCMK_Register_TypeDef;
+// The I2C address of our RTC
+#define RTCMK_ADDR (0x32)
 
 /*******************************************************************************
  *****************************   PROTOTYPES   **********************************
  ******************************************************************************/
 
+/* unix_timestamp is set when RTCMK_SetUnix is called, and then updated every
+ * second by an RTC interrupt. RTCMK_Unix_Now() is the preferred way of getting
+ * the current unix time.
+ */
+static inline time_t RTCMK_Unix_Now() { return unix_timestamp; }
```

Comment:
Functions in headers shouldn't be declared static unless you mean to duplicate them in every file that includes them

Reply:
Which I do because it's inline